bcc/module.go

```diff
@@ -135,34 +132,34 @@ func (bpf *Module) Close() {
 
 // LoadNet loads a program of type BPF_PROG_TYPE_SCHED_ACT.
 func (bpf *Module) LoadNet(name string) (int, error) {
```

Comment:
Should we expose log levels on these functions too?

Maybe setting logging is something rare enough that users should just use `Load()` explicitly.

Reply:
> Maybe setting logging is something rare enough that users should just use `Load()` explicitly.

I was leaning towards that. Also, in the current bcc release (0.5.0), enabling logging does not only enable verifier logging but also makes [bcc write hints to stderr](https://github.com/iovisor/bcc/blob/9634a002c694c7157fdff666522739e3eecb387b/src/cc/libbpf.c#L479-L481).

Reply:
Yeah, [I complained about that](https://github.com/iovisor/bcc/issues/1472). All good then.